### ISSUE TITLE
Not able to get Income cashflow quarterly data

### ISSUE THREAD

**Issue by baluinfo86** [quantbelt/ib_fundamental#12]
I tired to pull the income/cashflow quarterly data, getting below error message, but able to pull the eps_q, revenue_q data.

The fundamentals data for the security specified is not available.Missing reportType, contract: Stock(conId=41622169, symbol='FSLR', exchange='SMART', primaryExchange='NASDAQ', currency='USD', localSymbol='FSLR', tradingClass='NMS')
Traceback (most recent call last):

**Comment by gnzsnz** [comment]
please check #11 

IBKR API is been down for a few weeks now. There is nothing that i can do.

**Comment by baluinfo86** [comment]
ok thanks

**Comment by baluinfo86** [comment]
Did some debugging and found, all other method is working except income_quarter/income_annual/balance_quarter , basically anything that calls ReportsFinStatements. Wanted to check if it is working for you? just to make sure tws version is not causing any issue

**Comment by gnzsnz** [comment]
yes i have the same results, "ReportsFinStatements" is the one not returning results.

I had this issue in the past, and the outage was restored on IBKR side. I hope its fixed. But as said this is on IBKR side there is nothing that i can do.
